homework6/js/main.js

```diff
@@ -0,0 +1,81 @@
+//Написати функцію, яка приймає 4 аргументи і шукає найменше число серед них
+
+function minNumber(number1, number2, number3, number4) {
+        return Math.min(number1, number2, number3, number4);
+ }
+
+ minNumber(5, 8, 23, -5);
```

Comment:
<img width="243" alt="Screenshot 2022-09-20 at 21 42 12" src="https://user-images.githubusercontent.com/34886027/191338689-9f174d4a-a1fd-4db5-a1b1-4457a0af24dc.png">

Reply:
Remember - return returns a value which can be saved (assigned to a variable)
Consider this approach

```suggestion
 const result = minNumber(5, 8, 23, -5);
 // console.log(result) for test reasons
```